part-spec/sensor.json

```diff
@@ -0,0 +1,240 @@
+{
+    "$id": "https://github.com/chromeos/digital-datasheets/blob/main/part-spec/sensor.json",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "title": "sensor",
+    "oneOf": [
+        {
+            "title": "accelerometer",
+            "type": "object",
+            "required": [
+                "componentID"
+            ],
+            "properties": {
+                "componentID": {
+                    "description": "common component identifying information, such as mpn",
+                    "$ref": "definitions.json#/componentID"
+                },
+                "accelerometerType": {
+                    "description": "Type of accelerometer",
+                    "examples": [
+                        "capacitive MEMS",
+                        "piezoresistive",
+                        "piezoelectric"
+                    ],
+                    "type": "string"
+                },
+                "accelerationRanges": {
+                    "description": "range of force that accelerometer can measure",
+                    "examples": [
+                        "2",
+                        "4",
+                        "8",
+                        "16"
+                    ],
+                    "comment": "units of g",
+                    "type": "number"
+                },
+                "accelerationSensitivity": {
+                    "description": "smallest change in force the accelerometer is able to capture (typical). Corresponds to levels in acceleration ranges",
+                    "examples": [
+                        "16000",
+                        "8000",
+                        "4000",
+                        "2000"
+                    ],
+                    "comment": "units of LSB/g; conditions can include acceleration ranges and output resolution",
```

Comment:
I see LSB/g is a standard thing (I found the definition as the first result on Google), but maybe we want to define it here so that our spec can be more standalone. e.g. I'm thinking just:

"units of LSB/g   (least significant bits per g)

Reply:
fixed